pyoptsparse/pySNOPT/pySNOPT.py

```diff
@@ -664,27 +585,17 @@ def _set_snopt_options(self, iPrint, iSumm, cw, iw, rw):
         Set all the options into SNOPT that have been assigned
         by the user
         """
-
         # Set Options from the local options dictionary
         # ---------------------------------------------
         inform = np.array([-1], np.intc)
-        for name, value in self.set_options:
-            if name == "iPrint" or name == "iSumm":
+        for name, value in self.options.items():
+            # these do not get set using snset
+            if name in self.specialOptions or name in self.pythonOptions:
                 continue
 
             if isinstance(value, str):
-                if name == "Start":
```

Comment:
Don't you still need this part to set ``"Start"``?

Reply:
Nope, `Start` can be set in two places, either as an argument to `snkerc` (this is mandatory) or a separate option `Cold start` or `Warm start` etc passed to `snset` (this will overwrite the `snkerc` option). Previously, the value for `Start` would be used to set both which is redundant, I've removed that now.